ups/pipelines_lsst_io.table

```diff
@@ -56,6 +56,7 @@ setupRequired(pex_config)
 setupRequired(pex_exceptions)
 setupRequired(pipe_base)
 setupRequired(pipe_tasks)
+setupRequired(pipeline_tools)
```

Comment:
We cant have an lsst-dm repo here, nor do I think we should. This is supposed to be an optional tool that we run in a few cases and use as a guide post on what a longer term solution to this problem may be